src/ServiceControl.Monitoring.Data.Tests/RingBufferTests.cs

```diff
@@ -7,11 +7,14 @@
     public class RingBufferTests
     {
         RingBuffer ringBuffer;
+        IEntryProvider provider;
```

Comment:
Is `RingBuffer` still used anywhere in the tests?

Reply:
I'll be going through the tests right now. Let me back to this question shortly.

Reply:
What do you mean @WilliamBZA ? Is it tested somewhere? If yes, this is the class `RingBufferTests` for testing it. It's used in `RawDataReporterTests` as well to test the reporter component.

Does it answer your question?